packages/next/src/build/output/store.ts

```diff
@@ -49,26 +49,22 @@ function hasStoreChanged(nextStore: OutputState) {
 }
 
 let startTime = 0
+let trigger = ''
 
 store.subscribe((state) => {
   if (!hasStoreChanged(state)) {
     return
   }
 
   if (state.bootstrap) {
-    if (state.appUrl) {
-      Log.ready(`started server on ${state.bindAddr}, url: ${state.appUrl}`)
-    }
     return
   }
 
   if (state.loading) {
     if (state.trigger) {
-      if (state.trigger !== 'initial') {
-        Log.wait(`compiling ${state.trigger}...`)
-      }
-    } else {
-      Log.wait('compiling...')
+      trigger = state.trigger
+    } else if (trigger !== 'initial' && trigger) {
```

Comment:
Won't this always be `''` in this case? Having a hard time following what the condition is supposed to do.

Reply:
fixed in [6758a49](https://github.com/vercel/next.js/pull/54713/commits/6758a49291e52cf294daefe067ff01c7f8c58be0)